packages/@ember/debug/lib/deprecate.ts

```diff
@@ -170,13 +184,15 @@ if (DEBUG) {
     @param {String} message A description of the deprecation.
     @param {Boolean} test A boolean. If falsy, the deprecation will be displayed.
     @param {Object} options
-    @param {String} options.id A unique id for this deprecation. The id can be
+    @param {String} [options.id] A unique id for this deprecation. The id can be
       used by Ember debugging tools to change the behavior (raise, log or silence)
       for that specific deprecation. The id should be namespaced by dots, e.g.
       "view.helper.select".
-    @param {string} options.until The version of Ember when this deprecation
+    @param {string} [options.until] The version of Ember when this deprecation
```

Comment:
I think the brackets here infer that the value is optional, see http://yui.github.io/yuidoc/syntax/index.html#param.

```suggestion
    @param {string} options.until The version of Ember when this deprecation
```